src/BootstrapTable.js

```diff
@@ -142,7 +142,13 @@ class BootstrapTable extends Component {
     const { options, selectRow } = nextProps;
 
     this.store.setData(nextProps.data.slice());
-    let page = options.page || this.state.currPage;
+
+    let page;
+    if (options.page !== 'undefined') {
```

Comment:
I changed this: https://github.com/AllenFang/react-bootstrap-table/pull/447/commits/98499b2d3de0149f27e85a75e753d948e25d32a7